### PR TITLE
add gis.md documentation

### PR DESCRIPTION
**Pull request by ajodeh-juma**
---
name: New Config
about: A new cluster config
---

Please follow these steps before submitting your PR:

* [ ] If your PR is a work in progress, include `[WIP]` in its title
* [ ] Your PR targets the `master` branch
* [ ] You've included links to relevant issues, if any

Steps for adding a new config profile:
* [ ] Add your custom config file to the `conf/` directory
* [ ] Add your documentation file to the `docs/` directory
* [ ] Add your custom profile to the `nfcore_custom.config` file in the top-level directory
* [ ] Add your custom profile to the `README.md` file in the top-level directory
* [ ] Add your profile name to the `profile:` scope in `.github/workflows/main.yml`

### CODE REVIEW

**Review by ggabernet — APPROVED** [review]
Looks great!

**Comment by maxulysse** [comment]
@nf-core-bot fix linting

File: docs/gis.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+# Login to a compute node
+srun --pty bash
+
+## Load Nextflow and Singularity environment modules
+module purge
+source /mnt/projects/rpd/rc/init.2017-04
+module load miniconda3
+
+
+# Run a nextflow pipeline with dependencies bundles in a conda environment
```
Comment (apeltzer) [inline docs/gis.md:21]:
```suggestion
# Run a nextflow pipeline with dependencies bundled in a conda environment
```

**Review by apeltzer — CHANGES_REQUESTED** [review]

**Review by apeltzer — APPROVED** [review]

**Comment by apeltzer** [comment]
Going to ignore that one error here on the Nextflow installation for utd_sysbio as GHA seem to have quite some hiccups and it passed in the previous 2 CI Runs perfectly fine.

**Pull request merged**